app/models/profile.rb

```diff
@@ -196,6 +196,14 @@ def deactivate_due_to_gpo_expiration
     )
   end
 
+  def deactivate_due_to_ipp_expiration
```

Comment:
Might recommend writing tests for this in `spec/models/profile_spec.rb`

Reply:
See [554c8f6](https://github.com/18F/identity-idp/pull/11105/commits/554c8f6a9eb01d13bbb04c6490bf7ac691bc5e59).